deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml

```diff
@@ -39,3 +39,11 @@ spec:
     sourceName: opencloud-operators
     sourceNamespace: openshift-marketplace
     description: Operator for managing deployment of health check service.
+  - name: ibm-commonui-operator
+    namespace: ibm-commonui-operator
```

Comment:
Please change the namespace to `ibm-common-services`. All the common services will be deployed in the same namespace